ReportSimulationOutput/measure.rb

```diff
@@ -1802,6 +1870,11 @@ def report_timeseries_output_results(runner, timeseries_output_path, args, times
     else
       zone_temps_data = []
     end
+    if args[:include_timeseries_zone_conditions]
+      zone_conds_data = @zone_conds.values.select { |x| x.timeseries_output.sum(0.0) != 0 }.map { |x| [x.name, x.timeseries_units] + x.timeseries_output.map { |v| v.round(n_digits) } }
```

Comment:
Have you done a run and checked the outputs? The results are being rounded to `n_digits`, which is presumably fine for temperatures (since we have existing temperature outputs), but I wonder about the humidity outputs. Particularly humidity ratios, which are small numbers.

Reply:
This is a good point and one I noticed with the grouping of data with different units here. 

Does it make more sense to move the temp data from here to `include_timeseries_zone_temperatures` and change this one to `include_timeseries_zone_humidit(y/ies)`? We'd still have the issue of different units but could adjust without affecting the temps.

Reply:
I like how it's organized now. The most common request by far is to just get standard drybulb temperatures; everything beyond that feels like extra detail that most users won't want. And if we go the current route, existing users won't see any changes to their outputs.

Reply:
Sounds good, I'll keep it as is.

I can add a 4th digit to these if you think it's needed for the humidity ratio, which looks like it might be nice based on the results of the single unit test model below.

Reply:
Looking at the code again, we really shouldn't be using `n_digits` even to round temperature outputs. `n_digits` varies based on the run period and timeseries frequency and is really intended for energy use or loads (where you want higher resolution if you're just running, say, a single day). Ideally, I feel like we would define separate `n_digits_temp` and `n_digits_hr` with the desired resolution.

If you don't want to spend any more time on this, you're welcome to address my other comments and I can tackle this.

Reply:
Ok, I'll leave it for you to address in another PR.